subsys/bluetooth/audio/shell/bap_scan_delegator.c

```diff
@@ -441,9 +443,11 @@ static int cmd_bap_scan_delegator_sync_pa(const struct shell *sh, size_t argc,
 		return -ENOEXEC;
 	}
 
-	if (past_preference &&
-	    state->past_avail &&
-	    state->conn != NULL) {
+	if (0) {
+#if defined(CONFIG_BT_PER_ADV_SYNC_TRANSFER_SENDER)
+	} else if (past_preference &&
+		   state->past_avail &&
+		   state->conn != NULL) {
```

Comment:
Is there any reason `IS_ENABLED` cannot be used instead?

Reply:
Yes, there is a reference to a static function inside, which causes a build failure when using the IS_ENABLED, unfortunately